modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -34,9 +34,13 @@ If empty, the default value is `ingress.config.openshift.io/cluster` `.spec.doma
 If not set, the default value is based on `infrastructure.config.openshift.io/cluster` `.status.platform`:
 
 * Amazon Web Services (AWS): `LoadBalancerService` (with External scope)
+* Amazon Web Services (AWS): `loadBalancer.allowedSourceRanges`
```

Comment:
This is a list of default `endpointPublishingStrategy` default values. Instead, we can add this new API a few lines below under this line 
```
On GCP, you can configure the following endpointPublishingStrategy fields:
- loadBalancer.scope
- loadBalancer.allowedSourceRanges
```

Reply:
Addressed